tests/unit/gappsInstall.test.js

```diff
@@ -0,0 +1,276 @@
+        global.URL.createObjectURL.mockClear();
+        global.URL.revokeObjectURL.mockClear();
+
+        instance = new Instance({
+            fileUploadUrl: 'mocked'
+        });
+        plugin = new GAPPSInstall(instance, {
+            GAPPS_TITLE: 'TEST GAPPS TITLE',
+            DRAG_DROP_TEXT: 'TEST DRAG DROP TEXT',
+            UPLOAD_APK_TEXT: 'TEST UPLOAD APK TEXT',
+            BROWSE_BUTTON_TEXT: 'TEST BROWSE',
+            FILE_TOO_LARGE: 'TEST FILE TOO LARGE',
+            FILE_TYPE_NOT_APK: 'TEST FILE TYPE NOT APK',
+            FILE_SEND_FAILED: 'TEST FILE SEND FAILED'
+        });
+        initialView = plugin.instanciatedViews.get('InitialView');
+    });
+
+    afterEach(() => {
+        // Nettoyage des mocks après chaque test
+        jest.clearAllMocks();
+    });
+
+    describe('api', () => {
+        test('exposes a high level constructor', () => {
+            expect(typeof GAPPSInstall).toBe('function');
+        });
+
+        test('has a static name property', () => {
+            expect(GAPPSInstall.name).toBe('GAPPSInstall');
+        });
+    });
+
+    describe('view transitions', () => {
+        test('initial view shows correct content and translations', () => {
+            const container = document.getElementsByClassName('gm-gapps-plugin')[0];
+            expect(container.innerHTML).toEqual(expect.stringContaining('TEST GAPPS TITLE'));
+            expect(container.innerHTML).toEqual(expect.stringContaining('TEST DRAG DROP TEXT'));
+            expect(container.innerHTML).toEqual(expect.stringContaining('TEST UPLOAD APK TEXT'));
+            expect(container.innerHTML).toEqual(expect.stringContaining('TEST BROWSE'));
+        });
+
+        test('clicking install GAPPS button transitions to disclaimer view', () => {
+            initialView.installBtn.click();
+
+            const container = document.getElementsByClassName('gm-gapps-plugin')[0];
+            expect(container.innerHTML).toEqual(expect.stringContaining('DISCLAIMER'));
+            expect(container.innerHTML).toEqual(expect.stringContaining('GENYMOBILE SAS assumes no liability'));
+        });
+
+        test('clicking back button on disclaimer returns to initial view', () => {
+            // First go to disclaimer view
+            initialView.installBtn.click();
+
+            // Find and click back button
+            const backBtn = document.querySelector('.gm-disclaimer-actions .gm-btn:not(.gm-gradient-button)');
+            backBtn.click();
+
+            // Verify we're back to initial view
+            const container = document.getElementsByClassName('gm-gapps-plugin')[0];
+            expect(container.innerHTML).toEqual(expect.stringContaining('TEST GAPPS TITLE'));
+            expect(container.innerHTML).toEqual(expect.stringContaining('TEST DRAG DROP TEXT'));
+        });
+
+        test('clicking install button on disclaimer transitions to installing view', () => {
+            // First go to disclaimer view
```

Comment:
Thank you for these comments! Helps someone unfamiliar with the code base to understand the tests! 🙏🏻